generators/getMetric.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path')
+const fs = require('fs')
+const fsPromises = fs.promises
+
+const data = [
+  ['daily_avg_marketcap', 'Gets the daily average marketcap'],
```

Comment:
I don't see fiat currency and `_<number>y` and `_<number>d` suffixes as arguments.
Array of objects would be a better structure.